packages/flutter/test/cupertino/route_test.dart

```diff
@@ -439,6 +439,8 @@ void main() {
     await tester.pump(const Duration(milliseconds: 40));
     expect(tester.getTopLeft(find.byType(Placeholder)).dy, moreOrLessEquals(0.0, epsilon: 0.1));
 
+    await tester.pumpAndSettle();
```

Comment:
These are necessary additions because you now have to wait until transitions are over before you can interact with the next screen